stylelint-polaris/plugins/custom-properties-allowed-list/index.js

```diff
@@ -147,8 +147,11 @@ function validateCustomPropertyValues(allowedValues, prop, value) {
   return invalidValues;
 }
 
-module.exports.ruleName = ruleName;
-module.exports.messages = messages;
+module.exports = {
```

Comment:
This is changing the shape of what is exported. It used to be a function with some keys (yay prototypal inheritance) and now it's an object that contains a function. I'm pretty sure this isn't considered a valid stylelint rule anymore.

Reply:
Looking at `createPlugin` [source](https://github.com/stylelint/stylelint/blob/main/lib/createPlugin.js), I think we are safe. The `createPlugin` function returns an object with a rule and ruleName property. I've seen this export pattern used in other stylelint plugins as well 👍